CONTRIBUTING.md

```diff
@@ -1,5 +1,41 @@
-# Contribute
+# Contribute to the documentation
+We really appreciate your support!
 
-Thanks for your support!
-Please open an issue on github. More to come soon...
-~~Please open a pull request on GitHub!~~
+Please note that discussions should be kept in English. If you don't speak English fluently you can use a reasonably good online translator (e.g. [DeepL](https://www.deepl.com), [Reverso](https://www.reverso.net/text_translation.aspx), ...) which might be able to translate your language to English.
+
+Please have a look at the [issues page on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) and check if anybody has raised an issue with the same topic.
+
+If this is the case, please subscribe to the issue and explain your proposal.
+
+If that's not the case, please open a [new issue](https://github.com/jamulussoftware/jamuluswebsite/issues/new/choose), select which type of issue it is and describe your issue.
+
+If you want to open a Pull Request (after having described and discussed your proposal), keep in mind that your changes (if they are not just typos) might have to wait until everything is translated. We will bundle them (if everything is all right) with any other changes and upload all of them as a combined Pull Request to jamulus.io.
+
+Pull requests/issues for the website should have a title which looks like this:
+
+`[Tag (like Bug, Translation, etc.)][language code (e.g. en)] {short description of your changes}`
+
+## Branches, Translation and development process
+
+We currently use multiple branches to ensure that translators know what to translate. Therefore have a look at the guideline below.
+
+After having discussed and agreed on changes you want to be integrated to jamulus.io, you should ask yourself some questions:
+
+## 0. Is the language of your content already online and existing?
+No: Will discuss this later (since you introduce a new language).
```

Comment:
This No: " "Yes: " style is a bit weird, particularly as we don't use it in the section above. I'd keep it to "If not..." and "If so..." really. It's also slightly ambiguous: "**No:** A fix which doesn't need to be translated" could be taken to mean "**Don't** make a fix that doesn't need to be translated". It's just odd!